docs/site/development/updating-codes/update-language-script-info/language-script-description.md

```diff
@@ -7,7 +7,7 @@ title: Language Script Description
 The [`language\_script.tsv`](https://github.com/unicode-org/cldr/blob/main/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/language_script.tsv) data file should list all of the language / script combinations that are in common use. Usage by country is indicated in the [`country\_language\_population.tsv`](https://github.com/unicode-org/cldr/blob/main/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/country_language_population.tsv) spreadsheet.
 
 1. Every language should have 1 script considered the **primary** script.
-    1. This data is used to determine [the most Likely language and region](likelysubtags-and-default-content) so there needs to be at least 1 primary value.
+    1. This data is used to determine [the most Likely language and region](/development/updating-codes/likelysubtags-and-default-content) so there needs to be at least 1 primary value.
```

Comment:
```suggestion
    1. This data is currently used to determine [the likely script for a language](/development/updating-codes/likelysubtags-and-default-content) so there needs to be at least 1 primary value. Because it is the default, it determines the script of locales without language codes in the `<territoryInfo>`. 
```

See also https://unicode-org.atlassian.net/browse/CLDR-18155?focusedCommentId=178748